docs/src/tutorials/custom_impurity_density_pn_junction_lit.jl

```diff
@@ -0,0 +1,143 @@
+# # Advanced Example: Custom Impurity Profile
+
+# A very important input to the simulation is the impurity profile of the semiconductor.
+# It influences the electric potential and, thus, the electric field, capacitances,
+# drift paths and also the induced waveforms.
+
+# Arbitrary impurity density profiles can be defined and assigned to the semiconductor of the detector.
+# Also have a look into the manual: [Impurity Densities](@ref) and [Custom Impurity Density](@ref).
+
+# As an example, we define the impurity profile of a simple [p-n junction](https://en.wikipedia.org/wiki/P%E2%80%93n_junction)
+# and calculate the electric potential and depleted region and reproduce the 
+# typical [plot](https://en.wikipedia.org/wiki/P%E2%80%93n_junction#/media/File:Pn-junction-equilibrium-graphs.png)
+# shown in various books explaining p-n junctions.
+
+# For this, we use one of the example detectors: the infinite parallel plate capacitor in cartesian coordinates.
+# The plates span over `y` and `z`. It is made infinite by defining the world smaller than the contacts
+# and using reflecting boundaries in `y` and `z`.
+# Let's get started by loading the packages and the detector and have a look at its geometry:
+
+using Plots
+using SolidStateDetectors
+using Unitful
+
+sim = Simulation{Float32}(SSD_examples[:InfiniteParallelPlateCapacitor])
+plot(sim.detector)
```

Comment:
The plot looks very squeezed together. Is that because the world is much larger in `x` than the detector itself? Is this something we want to change?

![Parallel Plate Capacitor](https://user-images.githubusercontent.com/30291312/159172684-69384fd6-751e-4f22-91d8-93c96e265e44.png)

I think it would make sense to also set `show_semiconductor = true`, as this tutorial primarily deals with the semiconductor and not only the contacts.

Reply:
I changed some plotting parameters.